generator/generator_tests/feature_builder_test.cpp

```diff
@@ -6,11 +6,16 @@
 #include "generator/generator_tests_support/test_with_classificator.hpp"
 #include "generator/osm2type.hpp"
 
+#include "indexer/data_header.cpp"
 #include "indexer/classificator_loader.hpp"
 #include "indexer/feature_visibility.hpp"
+#include "generator/geometry_holder.hpp"
```

Comment:
please move this line to other generator includes

Reply:
done